.claude/hooks/claude-build-lock-guard.py

```
@@ -142,18 +168,68 @@ def leads_with_non_building_tool(command):
+
+
+def audit(decision, tool, command):
+    """Record an unlocked build next to the lock's own metrics. Never raises.
+
+    Best-effort by design: an audit failure must not block a tool call, and git is
+    only invoked on this rare path so the common case adds no latency.
+    """
+    try:
+        import subprocess
+
+        common = subprocess.run(
```

Ruff (0.16.1)

[warning] `subprocess.run` without explicit `check` argument

Add explicit `check=False`

(PLW1510)